lxd/db/cluster/update.go

```diff
@@ -107,6 +107,64 @@ var updates = map[int]schema.Update{
 	68: updateFromV67,
 	69: updateFromV68,
 	70: updateFromV69,
+	71: updateFromV70,
+}
+
+func updateFromV70(ctx context.Context, tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE groups (
```

Comment:
auth_groups to match the API and CLI auth prefix?
Also to disambiguate from other groups in lxd?

Reply:
or identity_groups?

Reply:
or identity_groups?

Reply:
I think `auth_groups` is good and means we can still have `identities_auth_groups`, `auth_groups_permissions`, and `auth_groups_identity_provider_groups`